core/tools/dnsmasq/bootstrap.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Exit if something goes wrong
+set -e
+
+# Dnsmasq provides DHCP and DNS servers for small networks
+echo "Installing dnsmasq."
+apt install -y --no-install-recommends dnsmasq
+
+echo "Configuring dnsmasq."
+CONFIG_ORIG_PATH="/home/pi/tools/dnsmasq/dnsmasq.conf"
+CONFIG_DEST_PATH="/etc/dnsmasq.conf"
+
+# Checks if the installation process indeed created the original file,
+# preventing changes in this behavior to silent affect our application.
+[ ! -f "${CONFIG_DEST_PATH}" ] && (
+    echo "Dnsmasq configuration file does not exist: ${CONFIG_DEST_PATH}"
+    exit 1
+)
+
+cp $CONFIG_ORIG_PATH $CONFIG_DEST_PATH
```

Comment:
This is not necessary since we can provide the path for the configuration file with `--conf-file` as an argument.